src/components/sidebar/cart/Cart.js

```diff
@@ -25,13 +27,17 @@ function Cart({ classes, sections }) {
 
   const label = uniqueSections.length === 1 ? 'Class' : 'Classes';
 
+
```

Comment:
I'm surprised linting didn't complain about the double empty lines but let's keep this at one.